packages/react-core/src/components/NotificationBadge/NotificationBadge.tsx

```diff
@@ -27,6 +30,8 @@ export interface NotificationBadgeProps extends Omit<ButtonProps, 'variant'> {
   isExpanded?: boolean;
   /** Determines the variant of the notification badge. */
   variant?: NotificationBadgeVariant | 'read' | 'unread' | 'attention';
+  /** Determines if the notification badge should animate on new notifications. */
+  hasAnimation?: boolean;
```

Comment:
During a call yesterday with Coker and Sarah, the name of the prop was sort of tossed around. Maybe something like `shouldAnimationTrigger`, `shouldNotify`, or something along those lines? Maybe worth tossing around before committing to anything.

Reply:
I think `hasAnimation` is probably fine, and `shouldAnimationTrigger` also sounds good to me, just a little wordier. I'd be fine with either. `shuoldNotify` sounds a little unclear in what behavior it changes to me though.

Reply:
i'd like to be part of that conversation, because neither of those alternatives are better imo than hasAnimation

Reply:
Upon further thoughts, `shouldAnimate` or `shouldNotify` both would be fine with me combined with the updated prop description.

Reply:
i went with `shouldNotify` and i'll just let it go :)